src/custom/pages/Profile/index.tsx

```diff
@@ -33,8 +35,19 @@ export default function Profile() {
             <Txt>
               <RefreshCcw size={16} />
               &nbsp;&nbsp;
-              <Txt secondary>Last updated:&nbsp;</Txt>
-              <strong>{lastUpdated || '-'}</strong>
+              <Txt secondary>
+                Last updated:&nbsp;
+                <MouseoverTooltipContent content="Data is updated on the background periodically.">
+                  <HelpCircle size={14} />
+                </MouseoverTooltipContent>
```

Comment:
Uber-minor-nit: I think it would be better to have the `?` before the `:`

```suggestion
                Last updated&nbsp;
                <MouseoverTooltipContent content="Data is updated on the background periodically.">
                  <HelpCircle size={14} />
                </MouseoverTooltipContent>:
```